content/aws/iam/assuming-roles.md

```diff
@@ -7,3 +7,32 @@ tags:
   - aws
   - assume-role
 ---
+
+# Assume Role via AWS Web Console
+
+**NOTE**: Due to the security implications, IAM policies are set up to **only** allow the root AWS account to assume roles into other accounts.
```

Comment:
Use the `dialog` shortcode and use the `important` type

Reply:
https://github.com/cloudposse/docs/pull/83/files#diff-7ca7eda112205749e7951ba4942ce13dR73